src/log4net.Tests/Core/LoggingEventTest.cs

```diff
@@ -138,4 +138,15 @@ static string ReviseThreadName(string? name)
     static void AssertIsCurrentThreadId(string name)
       => Assert.AreEqual(name, SystemInfo.CurrentThreadId.ToString(CultureInfo.InvariantCulture));
   }
+
+  [Test]
+  public void UserNameTest()
+  {
+    var expectedUserName =
```

Comment:
```suggestion
    string expectedUserName =
```
We switched away from var-Style some time ago.